code/forth/gforth/get

```diff
@@ -0,0 +1,11 @@
+#! /bin/sh -e
+# https://gforth.org/
+VER=0.7.3
+PREFIX="$PWD/gforth"
+wget -c https://ftp.gnu.org/gnu/gforth/gforth-$VER.tar.gz
+tar xzf gforth-$VER.tar.gz
+cd gforth-$VER
+./configure --prefix="$PREFIX"
+make
+make install
+
```

Comment:
I managed to build gforth locally and run the bbhw.fs with it on my macOS (Sonoma 14.5) by building also the gforth dependencies libffi and libtool, see the suggestion below.

```suggestion
GFORTH_VER=0.7.3
PREFIX="$PWD/gforth"

 # libffi: gforth dependency
wget -c https://github.com/libffi/libffi/releases/download/v3.4.6/libffi-3.4.6.tar.gz
tar xzf libffi-3.4.6.tar.gz
pushd libffi-3.4.6
./configure --prefix="$PREFIX" --disable-debug --disable-docs
make -j
make install
popd

# libtool: gforth dependency
wget -c https://ftpmirror.gnu.org/libtool/libtool-2.4.7.tar.gz
tar xzf libtool-2.4.7.tar.gz
pushd libtool-2.4.7
./configure --prefix="$PREFIX"
make -j
make install
popd

# gforth
wget -c https://ftp.gnu.org/gnu/gforth/gforth-$GFORTH_VER.tar.gz
tar xzf gforth-$GFORTH_VER.tar.gz
pushd gforth-$GFORTH_VER
LTDL_LIBRARY_PATH="$PREFIX/lib" ./configure --prefix="$PREFIX"
make -j
make install
popd
```

One possible caveat: I use Homebrew and I have gforth and all its dependencies installed using it. The installed versions seem to mess somewhat with the gforth configure script, which e.g. detects the installed libtool as glibtool and even that I have gforth installed.

Reply:
gforth is working on WSL 1 (Ubuntu 18.04) except for the which -s argument in the run script. pforth run script is not passing command line args, but the bbhw works otherwise as normal. Please fix these two issues and I will approve this PR. As to macOS I will just assume that anyone installing gforth via homebrew or MacPorts will be fine (I no longer use neither homebrew nor MacPorts because they pollute the global environment).

Reply:
The caveat I mention above is real: gforth seems to require gforth for bootstrapping its own build. All instructions for building it I can find point to obtaining a binary somewhere and making it accessible to the gforth configure script. Also libffi is probably not needed as the Homebrew-installed gforth seems to link to the system library `usr/lib/libffi.dylib`.
```shell
% otool -L $(which gforth)
/opt/homebrew/bin/gforth:
	/usr/lib/libffi.dylib (compatibility version 1.0.0, current version 30.0.0)
	/opt/homebrew/opt/libtool/lib/libltdl.7.dylib (compatibility version 11.0.0, current version 11.2.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1319.0.0)
```

Reply:
> As to macOS I will just assume that anyone installing gforth via homebrew or MacPorts will be fine (I no longer use neither homebrew nor MacPorts because they pollute the global environment).

I can confirm that the bbhw.fs works with Homebrew-install gforth 0.7.3 on macOS.

Reply:
The bootstrapping is for kernl{16,32,64}[lb].fi as far as I can tell. They are included with the release, but also rebuilt from source (kernel/main.fs) in the normal make process. A separate gforth is not required. 

pforth actually doesn't support command line arguments.